src/visual/Visualizer.js

```diff
@@ -86,6 +86,20 @@ export default class Visualizer {
     this._timer = null;
     this._step = null;
     this.clear();
+
+    const paperDefaultValidateConnectionMethod = this.paper.options.validateConnection;
```

Comment:
Thank you it is a nice note. The second proposed variant was selected. 
Done in 3571b0a491 commit